.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow deploys the master brance to the live site
+
+name: prod-deployment
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push events but only for the master branch
+  push:
+    branches: [ master ]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: Configure gcloud
+        run: gcloud config set compute/zone us-central1-c
+      - name: Run deploy script
+        run: |
+          gcloud compute ssh openreview@instance-matching-server --command '/bin/matcher_build.sh' --quiet
```

Comment:
Will we deploy to production every time we merge to master?

Reply:
Yes

Reply:
I believe this is not something that we want. We prefer to deploy first to the dev site and then manually to production in case there is something wrong in master

Reply:
I can change it to only manual.

Reply:
That would be great! Thanks